tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -149,7 +155,7 @@ Function PageReinstall
   ; and modify the messages presented to the user accordingly
   compare_version:
   StrCpy $R4 "$(older)"
-  ${If} $R5 == "wix"
+  ${If} $R7 == "wix"
```

Comment:
keep using `R5`

Reply:
$R5 is modified in the code below
https://github.com/tauri-apps/tauri/blob/46df2c9b917096388695f72ca4c56791fe652ef6/tooling/bundler/src/bundle/windows/templates/installer.nsi#L162-L185

Reply:
ah, that's fine then